### ISSUE TITLE
LR scheduling is used improperly

### ISSUE THREAD

**Issue by Erroler** [minar09/cp-vton-plus#69]
In the paper, the authors claim `The learning rate was first fixed at 0.0001 for 100K steps and then linearly decays to zero for the remaining steps`. 
However, the `step` method of the LR schedulers (declared on [1](https://github.com/minar09/cp-vton-plus/blob/master/train.py#L67) [2](https://github.com/minar09/cp-vton-plus/blob/916d7337703b49e18c8005f4b1847ba53c76c9e9/train.py#L132)) is never called, leading to a constant learning rate throughout training. 
I believe the fix is to add `scheduler.step()` on lines 105 and 172 of train.py (PyTorch documentation regarding LR schedulers can be found [here](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate)).
If ok by the authors I will create a PR addressing this issue.

**Comment by Erroler** [comment]
On a closer look, this issue also affects the [CP-VTON repository](https://github.com/sergeywong/cp-vton/blob/master/train.py).

**Comment by minar09** [comment]
Hi @Erroler , thank you very much for pointing it out. Sure, please go ahead and submit a PR. Thank you.
